src/docs/devices/Shelly-Pro-2/index.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Shelly Pro 2
+date-published: 2024-01-12
+type: relay
+standard: uk, eu
+board: esp32
+---
+
+Information about the pinout and internals of the Shelly Pro 2.
+
+It's the same board as the Shelly Pro 1. The only difference is the addition of a second relay.
+
+![Shelly Pro 2](shelly-pro-2.jpg)
+
+## Pinout
+
+ESP32 DOWDQ6| SN74HC595B | LAN8720A | Component
+------------|------------|----------|----------
+GPIO 4      |RCLK        |          |
+GPIO 13     |SER         |          |
+GPIO 14     |SRCLK       |          |
+GPIO 17     |            |CLKIN     |
+GPIO 18     |            |MDIO      |
+GPIO 19     |            |TXD0      |
+GPIO 21     |            |TXEN      |
+GPIO 22     |            |TXD1      |
+GPIO 23     |            |MDC       |
+GPIO 25     |            |RXD0      |
+GPIO 26     |            |RXD1      |
+GPIO 27     |            |CRS_DV    |
+GPIO 35     |            |          |Reset Button
+GPIO 36     |            |          |ADC Temperature
```

Comment:
I took the settings for the NTC temperature reading from Tasmota.
I tested them roughly and they seemed OK!

The NTC should normally be identical for the Shelly Pro 1. But I don't have that board so I can't be sure.
Once we are sure we could edit the Shelly Pro 1 page accordingly.